packages/db/lib/stackable.js

```diff
@@ -8,6 +10,14 @@ class DbStackable extends ServiceStackable {
     this.#updateConfig()
   }
 
+  async init () {
+    await super.init()
+    const itc = globalThis[kITC]
+    if (itc) {
+      globalThis[kITC].handle('getServiceMeta', this.getMeta.bind(this))
```

Comment:
IMHO we should not design a stackables in the way so they will have to overwrite some itc handers. This part should be hidden from a stackable interface.

Reply:
I agree. Anyway yes beyond the scope of this PR. We can revisit later.

Reply:
1. I don't think it should be a part of meta keys.
2. I don't like the idea of "let's write some bad code in the PR and fix it later". It will never be fixed in this way.

Reply:
1. Why not? We can't keep adding new methods to the Stackable API for every new need. Metadata are perfectly fit for this.
2. My take is that Marco has limited knowledge of our refactoring so I was trying to unlock him and me/you will take care of this in a short future.